src/AjaxUploader.tsx

```
@@ -21,121 +22,79 @@ interface ParsedFileInfo {
+    };
+  }, [abort]);
 
   /**
    * Process file before upload. When all the file is ready, we start upload.
    */
-  processFile = async (file: RcFile, fileList: RcFile[]): Promise<ParsedFileInfo> => {
-    const { beforeUpload } = this.props;
-
+  const processFile = async (file: RcFile, fileList: RcFile[]): Promise<ParsedFileInfo> => {
+    const { beforeUpload } = props;
     let transformedFile: BeforeUploadFileType | void = file;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)